tsconfig.json

```diff
@@ -1,31 +1,25 @@
 {
   "compilerOptions": {
-      "target": "es2015",
-      "module": "ESNext",
-      "moduleResolution": "Node",
-      "strict": false,
-      "noImplicitAny": true,
-      "sourceMap": false,
-      "noUnusedLocals": true,
-      "noUnusedParameters": true,
-      "strictNullChecks": true,
-      "strictFunctionTypes": true,
-      "declaration": true,
-      "baseUrl": "./src",
-      "lib": [
-        "dom",
-        "es2016",
-        "es2017.object"
-      ],
-      "outDir": "dist",
-      "skipLibCheck": true,
-      "esModuleInterop": true
+    "target": "es2022",
+    "module": "ESNext",
+    "moduleResolution": "Bundler",
+    "strict": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "lib": [
+      "es2022"
+    ],
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "noEmit": true
   },
   "include": [
-    "src/**/*.ts"
+    "src/**/*.ts",
   ],
-  "exclude": [
-      "node_modules",
-      "**/*.spec.ts"
+  "exclude": [],
+  "references": [
+    {
+      "path": "./tsconfig.tests.json"
```

Comment:
This is now referenced in the root tsconfig, meaning both can be run simultaneously.